variables.tf

```diff
@@ -0,0 +1,110 @@
+variable "cluster_arn" {
```

Comment:
could you make a choice in orderint e.g. description - type - default - validation and make thsi consistent accross the variables